docs/fidesops/docs/guides/configuration_reference.md

```diff
@@ -52,6 +52,7 @@ The `fidesops.toml` file should specify the following variables:
 |`TASK_RETRY_DELAY` | `FIDESOPS__EXECUTION__TASK_RETRY_DELAY` | int | 20 | 5 | The delays between retries in seconds
 |`TASK_RETRY_BACKOFF` | `FIDESOPS__EXECUTION__TASK_RETRY_BACKOFF` | int | 2 | 2 | The backoff factor for retries, to space out repeated retries.
 |`REQUIRE_MANUAL_REQUEST_APPROVAL` | `FIDESOPS__EXECUTION__REQUIRE_MANUAL_REQUEST_APPROVAL` | bool | False | False | Whether privacy requests require explicit approval to execute
+|`MASKING_STRICT` | `FIDESOPS__EXECUTION__MASKING_STRICT` | bool | True | True | Mask via updates only.  If False, can use defined deletes or GDPR endpoints, which can can delete data beyond what is specified on the Policy.
```

Comment:
Would be great to clarify that if masking strict is true, you'll need an update endpoint, but if masking strict is false, you'll need at least one of the 3 endpoint types (update/gdpr/delete)